Source/JavaScriptCore/runtime/ThrowScope.h

```diff
@@ -65,9 +63,6 @@ class ThrowScope : public ExceptionScope {
 #define DECLARE_THROW_SCOPE(vm__) \
     JSC::ThrowScope((vm__), JSC::ExceptionEventLocation(EXCEPTION_SCOPE_POSITION_FOR_ASAN(vm__), __FUNCTION__, __FILE__, __LINE__))
 
-#define throwScopePrintIfNeedCheck(scope__) \
```

Comment:
I believe this is a debugging tool. Since it is already #def'd out, do we need to remove it? @MenloDorian would know this for sure, I will ping him

Reply:
This is fine to remove.  Thanks.